### DISCUSSION TITLE
htmx 1.9.9 and different behaviors with DOMContentLoaded event

### DISCUSSION THREAD

**Discussion by CleitonDeLima** [bigskysoftware/htmx#2033] [Q&A] [answered]
The behavior of the following code is behaving differently than the previous version of htmx:

htmx 1.9.8
```js
document.addEventListener("DOMContentLoaded", () => {
  htmx.onLoad(() => console.log('fired!'))
});

>>> fired!
```

htmx 1.9.9
```js
document.addEventListener("DOMContentLoaded", () => {
  htmx.onLoad(() => console.log('fired!'))
});

>>> 
```

In version 1.9.8 `htmx.onLoad` was called the first time the page was loaded, in version 1.9.9 this no longer happens, is this a bug? Is it ideal to change `DOMContentLoaded` to `htmx.onLoad` as well?

**Comment by Telroshan** [comment, marked as answer]
Hey, it's indeed a bug, thanks for the report
Submitted a fix for it in #2037 
Don't hesitate to open issues instead of discussions when you encounter problems like this btw!
